code/iaas/healthcheck/src/main/java/io/cattle/iaas/healthcheck/process/ServiceEventCreate.java

```diff
@@ -22,15 +24,20 @@
 
 import java.util.Arrays;
 import java.util.List;
+import java.util.Date;
 
 import javax.inject.Inject;
 import javax.inject.Named;
 
 
+
 @Named
 public class ServiceEventCreate extends AbstractObjectProcessHandler implements ProcessHandler, Priority {
 
     private static final List<String> UP_STATES = Arrays.asList(CommonStatesConstants.ACTIVE, CommonStatesConstants.UPDATING_ACTIVE);
+    private static final Long SECOND_MILLIS = 1000L;
+    private static final long STAY_ACTIVE_SECONDS = ArchaiusUtil.getLong("stay_active.seconds").getValue() * SECOND_MILLIS;
```

Comment:
Should we name it with correlation with service.lastActive parameter?

Reply:
lastActiveTime = time service was last active 
Changing it to, STAY_ACTIVE_MILLISECONDS = duration network service needs to stay active for before network's considered Up. 
Does that sound ok?